### ISSUE TITLE
Bug Report: skip zero value from Volumes

### ISSUE THREAD

**Issue by kuhytitusdlz** [matplotlib/mplfinance#28]
**Describe the bug**
if one of volume = 0
```
2020-02-12  54.41  56.36  54.39  55.79  305454      USD
2020-02-13  55.76  56.84  54.97  56.45       0      USD
```
then i get error when run:
```
    apds = [mpf.make_addplot(moexfl['non_com'], scatter=True, color='g', secondary_y='auto')]
    mpf.plot(stock_df, addplot=apds, type='candle', figratio=(25.6, 14.4),figscale=0.5, volume=True)
```
```
header: Content-Security-Policy: upgrade-insecure-requests; block-all-mixed-content
Traceback (most recent call last):
  File "C:\PycharmProjects\DataAnalysis\venv\lib\site-packages\mplfinance\plotting.py", line 34, in decorator
    return func(*args, **kwargs)
  File "C:\PycharmProjects\DataAnalysis\venv\lib\site-packages\mplfinance\plotting.py", line 352, in plot
    lo = math.log(math.fabs(min(volumes)),10) - 0.5
ValueError: math domain error
Backend TkAgg is interactive backend. Turning interactive mode on.
```

**Comment by DanielGoldfarb** [comment]
Thank you very much for reporting this issue.

I have fixed the crash; the pull request is currently processing and should be available on PyPi sometime tomorrow.  

I don't want to to "skip zero values" as the title of this issue suggest.  It is certainly a bug to crash on zero values, and that has been fixed.  To explain: the purpose of mplfinance is the "visualization and visual analysis of financial data."  I don't feel it is within the purvue of mplfinance to make decisions about which data to display, nor to provide tools to filter or clean up the data.  There are simple ways, using Pandas, to either delete rows with any values that are zero, or if you prefer, to interpolate, or something else.

I have tested the fix, and it will not crash, but _will_ display a zero volume for the trade date that has a zero volume in the data.  HTH. 

I will leave this issue Open until the package is published to PyPi.
